tools/linter_lib/custom_check.py

```diff
@@ -511,10 +535,12 @@ def build_custom_checkers(by_lang):
          'exclude': set(['static/templates/settings/display-settings.handlebars',
                          'templates/zerver/keyboard_shortcuts.html',
                          'templates/zerver/markdown_help.html']),
-         },
+         'good_lines': ['<img src="{{source_url}}" alt="{{name}}" />', '<img alg="" />'],
+         'bad_lines': ['<img alt="Foo Image" />']},
         {'pattern': '\Walt=["\']{{ ?["\']',
          'description': "alt argument should be enclosed by _().",
-         },
+         'good_lines': ['<img src="{{source_url}}" alt="{{name}}" />'],
```

Comment:
As described above, alt argument should be enclosed by _(). Do `git grep 'alt="\{\{'`.